### DISCUSSION TITLE
personalize owncast's favicon

### DISCUSSION THREAD

**Discussion by roughnecks** [owncast/owncast#3481] [Q&A]
Hello, is there a way of doing that?
We're running a docker instance.

Thanks

**Comment by gabek** [comment]
There's no supported way. You might be able to change it within the browser by targeting the favicon's DOM element in Javascript and swapping it out, but I'm not sure. And that wouldn't work outside a Javascript environment.

You could also rebuild the source, swapping out any assets you'd prefer.

**Comment by roughnecks** [comment]
I see, thanks.
We watched a handful of streaming channels and they all had the default favicon, all but one! So we thought there should be a way but rebuilding everything looks overkill to me.

**Reply by gabek** [reply]
You should ask them how they did it!

**Comment by roughnecks** [comment]
Eheh, I asked in their chat but someone replied to me the owner wasn't there and to ask on discord, but I would prefer not to 🙂
